state-chain/runtime/src/migrations/solana_integration.rs

```diff
@@ -91,7 +91,7 @@ impl OnRuntimeUpgrade for SolanaIntegration {
 				cf_runtime_upgrade_utilities::genesis_hashes::SISYPHOS => [0u8; 32].into(),
 				_ => {
 					// Assume testnet
-					todo!()
+					[0u8; 32].into()
```

Comment:
do we not have the contract on testnets?

Reply:
i dont think so. @albert-llimos ?

Reply:
No. Contracts are not finalized yet.